<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser_text">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_new">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>字体管家棉花糖</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>新建文件~</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_old">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>字体管家棉花糖</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择文件~</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪橄榄体简</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>文件名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪橄榄体简</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>所剩余金额：</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_less">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_check">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>字体管家棉花糖</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>点我查看~</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪橄榄体简</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>想要增加的金额：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_getmoney">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪橄榄体简</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>产生流水原因：</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_Reason">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>311</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_time">
    <property name="geometry">
     <rect>
      <x>529</x>
      <y>315</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方圆卡通POP字</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2020</year>
      <month>7</month>
      <day>2</day>
     </datetime>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>270</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪橄榄体简</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设置时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>字体管家棉花糖</family>
     </font>
    </property>
    <property name="text">
     <string>PS：增加为正，减少为负，支持两位小数哟~</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sure">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>490</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>我弄好啦，搞起来~</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>字体管家棉花糖</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>记账小帮手</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>651</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>字体管家棉花糖</family>
     </font>
    </property>
    <property name="text">
     <string>PS:由于输出格式固定，若未使用过此软件，建议先点击“新建文件”建立文件之后，再添加流水</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
